supabase/migrations/16_tenants.sql

```diff
@@ -4,7 +4,17 @@ create table tenants (
 
   -- TODO(johnny): In the future, we expect to hang billing
   -- and data-plane assignment onto this record.
-  tenant      catalog_tenant unique not null
+  tenant                  catalog_tenant unique not null,
+
+  captures_quota          integer        not null DEFAULT 10,
```

Comment:
supernit: we use lowercase keywords. See comment at the top of the first migration `01_json.sql` for details on style.

Reply:
done!